app/components/media/image_component.html.erb

```diff
@@ -0,0 +1,3 @@
+  <div class="flex sm:col-span-2 justify-center <%= classes %>">
+    <%= image_tag @media[:image_path], alt: @media[:image_alt], class: 'w-20 h-20' %>
+  </div>
```

Comment:
Linter: Final newline

Reply:
Addressed this in https://github.com/TheOdinProject/theodinproject/pull/3926/commits/db8e1fe866fa7c008677db7ef08b1ca782842597